src/lib/secrets/wallets.ml

```diff
@@ -48,16 +48,22 @@ let load ~logger ~disk_location : t Deferred.t =
   in
   {cache; path}
 
-(** Effectfully generates a new private key file and a keypair *)
-let generate_new t : Public_key.Compressed.t Deferred.t =
-  let keypair = Keypair.create () in
-  let privkey_path = get_path t (Public_key.compress keypair.public_key) in
+(** Effectfully generates a new private key file for the given keypair *)
```

Comment:
Does "effectfully" mean something here, beyond the file generation? Not sure it's helpful.